config/routes.rb

```diff
@@ -1,7 +1,12 @@
 Rails.application.routes.draw do
-  get 'crud', to: "crud#index"
+  get 'crud', to: "employee_crud#index"
+  get 'crud/:id', to: "employee_crud#index"
   get 'admin', to: "home#index"
   post 'admin', to: "home#login"
+  post 'logout', to: "employee_crud#logout"
+  post 'createEmployee', to: "employee_crud#create"
```

Comment:
In ruby, it's preferred to use snake-case rather than camelcase. It's just a convention and there is no harm on doing so, but I'd recommend following the conventions, so you get used to them 😄